### DISCUSSION TITLE
v1.0.0-M34

### DISCUSSION THREAD

**Discussion by rossabaker** [http4s/http4s#6515] [Releases]
This release is the latest milestone in the 1.x series, and is _not_ binary compatible with previous 1.0 milestones.  It also includes the changes from v0.23.13.

### http4s-core
* Remove `CollectionCompat` by @danicheg in https://github.com/http4s/http4s/pull/6508
### http4s-server
* Address the entity model in the server's `RequestLogger` by @danicheg in https://github.com/http4s/http4s/pull/6323
* Fix regression of `Router#define` by @danicheg in https://github.com/http4s/http4s/pull/6371
* Relax `server.middleware.Caching` constraints by @danicheg in https://github.com/http4s/http4s/pull/6490
### http4s-client
* Take into account the `Entity` model in the `ResponseLogger` by @danicheg in https://github.com/http4s/http4s/pull/6319
### Behind the scenes
* Merge 0.23 -> main by @armanbilge in https://github.com/http4s/http4s/pull/6430
* vault, scala-java-time updates via 0.23 by @armanbilge in https://github.com/http4s/http4s/pull/6443
* Ignore cats updates on main by @armanbilge in https://github.com/http4s/http4s/pull/6472
* 0.23 -> main by @armanbilge in https://github.com/http4s/http4s/pull/6499

**Full Changelog**: https://github.com/http4s/http4s/compare/v1.0.0-M33...v1.0.0-M34

**Comment by rossabaker** [comment]
Tracking releases of the ancillary repos.  Modules where a volunteer maintainer steps up cut to the front of the line! :smile:

* [ ] armeria
* [X] [blaze v1.0.0-M34](https://github.com/http4s/blaze/releases/tag/v1.0.0-M34)
* [ ] boopickle
* [x] [dom v1.0.0-M34](https://github.com/http4s/http4s-dom/releases/tag/v1.0.0-M34)
* [ ] fabric
* [x] [feral v1.0.0-M4](https://github.com/typelevel/feral/releases/tag/v1.0.0-M4)
* [ ] finagle
* [X] [jdk-http-client v1.0.0-M3](https://github.com/http4s/http4s-jdk-http-client/releases/tag/v1.0.0-M3)
* [ ] jetty
* [x] [netty v0.6.0-M2](https://github.com/http4s/http4s-netty/releases/tag/v0.6.0-M2)
* [ ] okhttp-client
* [ ] play-json
* [x] [prometheus-client v1.0.0-M34](https://github.com/http4s/http4s-prometheus-client/releases/tag/v1.0.0-M34)
* [ ] rho
* [X] [scala-xml v1.0.0-M34](https://github.com/http4s/http4s-scala-xml/releases/tag/v1.0.0-M34)
* [x] [scalatags v1.0.0-M34](https://github.com/http4s/http4s-scalatags/releases/tag/v1.0.0-M34)
* [X] [servlet v1.0.0-M34](https://github.com/http4s/http4s-servlet/releases/tag/v1.0.0-M34)
* [ ] session
* [ ] tomcat
* [x] [twirl v1.0.0-M34](https://github.com/http4s/http4s-twirl/releases/tag/v1.0.0-M34)

**Comment by FrancescoSerra** [comment]
done! https://github.com/http4s/http4s-twirl/releases/tag/v1.0.0-M34
